core/src/main/java/org/eclipse/hono/config/ProtocolAdapterProperties.java

```diff
@@ -23,6 +23,7 @@
     private boolean jmsVendorPropsEnabled = false;
     private boolean defaultsEnabled = true;
     private long eventLoopBlockedCheckTimeout = 5000L;
+    private long concurrentConnectionsLimit = Long.MAX_VALUE;
```

Comment:
I think an int will be more than enough, don't you think?

Reply:
Right. With 0 as "unlimited"?

Reply:
or `Integer.MAX_VALUE`?

Reply:
`Integer.MAX_VALUE` is fine.

I just remembered where the `Long` comes from: I blame the class `MicrometerBasedMetrics` where open connections are counted with `AtomicLong` :)